.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,56 @@
+# e2e test for nginx helper plugin
+
+name: CI for Login with nginx helper test
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: 
+      - develop
+      - master
+      - automation
```

Comment:
@pavanpatil1 Remove automation branch, i think there are only 2 main branches develop and master